HW04/anastasia_evsihneyeva/lib/student.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Student
+  attr_reader :name, :surname
+  attr_accessor :all_hw, :hw_in_progress, :notification_list
+
+  def initialize(name:, surname:)
+    @name = name
+    @surname = surname
+    @all_hw = []
+    @notification_list = []
+  end
+
+  def notifications
+    notification_list.each(&:output)
+  end
+
+  def notify(notification)
+    notification_list << notification
+  end
+
+  def mark_as_read!
+    notification_list.clear
+  end
+
+  def homeworks
+    all_hw.each do |hw|
+      puts "#{hw.title} - #{hw.description}" if hw.status != 'Finished'
+    end
+  end
```

Comment:
```suggestion
```
You have all_hw and you can return it and see the data, you don't need to add output of some data from puts to the business logic, you can do it in app.rb if necessary